### ISSUE TITLE
Access for BAH team members to GIDS Production

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-team#8897]
# Description
- Please provide your Contracting Officer's Representative (COR) name.
   > Doris Lin
- Please provide your name.
   > Desiree Turner
- Please Provide your email.
   > turner_desiree@bah.com or desiree.turner@va.gov
- Please provide the name of your team, your role on that team, and the name of the company you work for.
   > Booz Allen ("BAH") Team; We work for Booz Allen Hamilton
- Requesting Production Access to GIDS for the following individuals in order to support our testing efforts. If possible, we would be fine with read-only access but understand that may not be an option right now. 
   > Michael Delli-Gatti; Delli-Gatti_Michael@bah.com; Michael.Delli-Gatti@va.gov; QA Engineer; @delli-gatti-michael 
   > Adam Sonntag; Sonntag_Adam@bah.com; Adam.Sonntag@va.gov; QA Engineer; @sonntag-adam 
   > Dan Shawkey; Shawkey_Daniel@bah.com; Daniel.Shawkey@va.gov; Technical Lead; @shawkey-daniel 
   > Devin McCurdy; McCurdy_Devin@bah.com; Devin.McCurdy@va.gov; Senior Developer; @dmccurdybah 
   > Matt Roth; roth_matthew@bah.com; matthew.roth@va.gov; Senior Developer; @zurbergram

**Comment by desireeturner** [comment]
Request for GIDS Production access. @department-of-veterans-affairs/vsp-operations cc @jholton

**Comment by jholton** [comment]
cc: @gunsch BAH team's GIDS production access request

**Comment by gunsch** [comment]
cc @KevinMHoffmanUSDS @drorva 

Some clarifying of VSP's stance since GIDS operations still live in a muddled gray area between VSP and BAH teams:

Prod database access grants BAH team more ability to break production (specifically for education-related tooling, presumably not for anything else). If there are application issues in production because of DB issues connected to this elevated access, VSP will still provide support but will expect BAH to handle directly solving the issue.

To be clear: I'm supportive of granting this access, and hope it's helpful in getting BAH team some more direct control over their application 👍

**Comment by desireeturner** [comment]
Any updates on this to report to the BAH team? Thanks @gunsch @KevinMHoffmanUSDS @drorva !

**Comment by drorva** [comment]
Sorry it fell of the radar. @gunsch let's please grant this access. Can we provide, read-only access as suggested? If not, let's provide the access you mention "break production (specifically for education-related tooling, presumably not for anything else)."

**Comment by jsev-io** [comment]
Hello all, just to update, I'm currently looking at getting a solution in place for this, and will update as I make progress! 
Thanks,

**Comment by desireeturner** [comment]
@jsev-io Any updates on this? Thank you!

**Comment by jsev-io** [comment]
@desireeturner 
Just wanted to update, I've made much headway with this proocess, though I completely abandoned using DMS. Instead, I've gotten metadata mapping from source to target functional, and ETL scripts generating using glue.
 Now, just working on the actual ETL, and resolving any IAM perm conflicts that arise between the Glue service, and the bucket.
I am tweaking the ETL Job I created to include all of your tables in the DB, as right now, I'm just testing with 1 table per job. I did have a successful response from the service though.
Now I just want to validate we can query properly with Athena, and we should be good. I'm aiming at having this fixed next week.
